src/Translator/Collection.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace TCG\Voyager\Translator;
+
+use Illuminate\Support\Collection as IlluminateCollection;
+
+class Collection extends IlluminateCollection
+{
+    //
```

Comment:
Custom macros can be added only to the collection of the translator.

And I will be going to add some later.